Makefile.helper.mk

```diff
@@ -2,13 +2,11 @@
 # SRO-specific options
 
 SPECIALRESOURCE  ?= driver-container-base
-NAMESPACE        ?= openshift-special-resource-operator
+NAMESPACE        ?= special-resource-operator
 PULLPOLICY       ?= IfNotPresent
 TAG              ?= $(shell git rev-parse --abbrev-ref HEAD)
 CSPLIT           ?= csplit - --prefix="" --suppress-matched --suffix-format="%04d.yaml"  /---/ '{*}' --silent
 YAMLFILES        ?= $(shell  find manifests charts -name "*.yaml"  -not \( -path "charts/lustre/lustre-aws-fsx-0.0.1/csi-driver/*" -prune \)  -not \( -path "charts/*/shipwright-*/*" -prune \) -not \( -path "charts/experimental/*" -prune \) )
-PLATFORM         ?= ""
-SUFFIX           ?= $(shell if [ ${PLATFORM} == "k8s" ]; then echo "-${PLATFORM}"; fi)
```

Comment:
Same here, I think we should keep it but change the default behaviour ocp --> k8s

Reply:
I don't see any use case for generating identical manifests in a different directory for another platform - do you?

Reply:
I do, I know it is ugly but this is how `kustomize` overlays works (at least with the given structure generated by `operator-sdk` )